src/engine/video/image.cpp

```diff
@@ -1267,7 +1264,8 @@ void AnimatedImage::Draw(const Color &draw_color) const
     _frames[_frame_index].image.Draw(blended_color);
 }
 
-bool AnimatedImage::Save(const std::string &filename, uint32_t grid_rows, uint32_t grid_cols) const
+bool AnimatedImage::Save(const std::string &filename,
```

Comment:
For next time, when you see `<type> &<var>`, could you take the opportunity to change it into `<type>& <var>`? Thanks.